react-native-libraries.json

```
@@ -12124,5 +12124,24 @@
     "expoGo": true,
     "dev": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/adobe/aepsdk-react-native",
```

Hey @blumk, this fail is correct. In monorepos `githubUrl` needs to point to the directory, where given package resides:

```suggestion
    "githubUrl": "https://github.com/adobe/aepsdk-react-native/tree/main/packages/core",
```